app/src/main/java/com/sun/premierleague/ui/team/TeamPresenter.kt

```diff
@@ -0,0 +1,54 @@
+package com.sun.premierleague.ui.team
+
+import com.sun.premierleague.data.model.Coache
+import com.sun.premierleague.data.model.Player
+import com.sun.premierleague.data.model.TeamItem
+import com.sun.premierleague.data.source.FootballRepository
+import com.sun.premierleague.data.source.remote.OnDataLoadedCallback
+import java.lang.Exception
+
+class TeamPresenter(
+    private val view: TeamContact.View,
+    private val repository: FootballRepository
+) : TeamContact.Presenter {
+    var idTeam: String? = null
+
+    override fun start() {
+        idTeam?.let { getTeamInformation(it) }
+    }
+
+    override fun getTeamInformation(idTeam: String) {
+        repository.getTeam(idTeam, object : OnDataLoadedCallback<List<TeamItem>> {
+            override fun onSuccess(data: List<TeamItem>) {
+                val coaches: List<Coache> = data[0].coaches
+                val players: List<Player> = data[0].players
```

Comment:
ĐOạn này phải check data có empty không đã 
```suggestion
                val coaches = data[0].coaches
                val players = data[0].players
```